gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -7711,27 +7715,28 @@ components:
         ###############################
         # Other Query Parameters      #
         ###############################
-        apisSearchExpandsParam:
+        apisGetExpandsParam:
             name: expands
             in: query
-            description: Expansion of data to return in APIs.
+            description: Expansion of data to return in APIs for GET operation.
             schema:
                 type: array
                 items:
                     type: string
                     enum:
                         - deploymentState
-        apisGetExpandsParam:
+                        - primaryOwner
```

Comment:
Don't move "primary owner", it is used in the search resource, so it should stay in the search query param.